src/js/SignUpUI.js

```diff
@@ -0,0 +1,43 @@
+import Auth from './AuthUI';
+import AuthAPI from './AuthAPI';
+import SignIn from './SignInUI';
+
+export default class SignUp extends Auth {
+  static user = document.querySelector('#user-up');
+
+  static pass = document.querySelector('#pass-up');
+
+  static email = document.querySelector('#email-up');
+
+  static signUp() {
+    const { user, pass, email } = SignUp;
+    if (user.value.length < 3)
+      Auth.error.innerText = `Login at least three letter.`;
+    else if (!Auth.regExpPass.test(pass.value))
+      Auth.error.innerText = `Password must contain at least one letter, at least one number, and be longer than six charaters.`;
+    else if (!Auth.regExpEmail.test(email.value))
+      Auth.error.innerText = `Incorrect email.`;
```

Comment:
I think that this is bad validation practice

Reply:
why?

Reply:
I think validation should be a separate method